hazelcast/src/main/java/com/hazelcast/multimap/impl/operations/MultiMapOperationFactory.java

```diff
@@ -24,8 +25,7 @@
 
 import java.io.IOException;
 
-public class MultiMapOperationFactory
-        implements OperationFactory {
```

Comment:
For the same reasons as the Abstract Map OperationFactory, please introduce an AbstractMultiMapOperationFactory that can take care of the repeating noise.